BepressImportDom.inc.php

```diff
@@ -270,8 +274,19 @@ function _handleArticleNode() {
 
 		// Set copyright year and holder and license permissions
 		$copyrightYear = date("Y", strtotime($articlePublicationDate));
-		// FIXME: `Submission::getAuthorString()` deprecated in 3.2.0.0
-		$copyrightHolder = $this->_submission->getAuthorString();
+
+		// Re-fetch publication with updated author data
+		/** @var Publication $publication */
+		$publication = $publicationDao->getById($publication->getId());
+		$authorUserGroupIds = array_map(function($author) {
+			return $author->getData('userGroupId');
+		}, $publication->getData('authors'));
```

Comment:
I've just fixed a bug in upstream where the `getData('authors')` is returning a generator, which was being casted as an array (and failing), so moving to a `foreach` will save debugging time in the future 👀